Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -0,0 +1,272 @@
+        }
+
+        private void SetupButtonCallbacks()
+        {
+            _container.Q<Button>("UserProfilePageAccountCardNoAccountButton").RegisterCallback<ClickEvent>(_ => OpenLink(""));
+            _container.Q<Button>("UserProfilePageBootstrapAnotherProfileButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                var targetWizard = _container.Q<VisualElement>("AddNewProfile");
+                ChangeWizard(targetWizard);
+            });
+            _container.Q<Button>("UserProfilePageAccountNewProfileCreateButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                if (SaveProfile())
+                {
+                    var targetWizard = _container.Q<VisualElement>("BootstrapMenu");
+                    ChangeWizard(targetWizard);
+                }
+                else
+                {
+                    Debug.Log("Error");
+                }
+            });
+            _container.Q<Button>("UserProfilePageAccountNewProfileCancelButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                ClearCredentials();
+                SetupBootMenu();
+            });
+            _container.Q<Button>("UserProfilePageBootstrapStartButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                BootstrapSettings.RefreshBucketName();
+                OpenS3Popup(BootstrapSettings.BucketName);
+            });
+            _container.Q<Button>("UserProfilePageBootstrapAnotherBucketButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                OpenS3Popup(_stateManager.BucketName);
+            });
+            _container.Q<Button>("AccessKeyToggleReveal").RegisterCallback<ClickEvent>(_ =>
+            {
+                var accessToggle = _container.Q<TextField>("AccessKeyField");
+                ToggleHiddenText(accessToggle);
+            });
+            _container.Q<Button>("SecretKeyToggleReveal").RegisterCallback<ClickEvent>(_ =>
+            {
+                var secretToggle = _container.Q<TextField>("SecretKeyField");
+                ToggleHiddenText(secretToggle);
+            });
+            _container.Q<Button>("AddProfile").RegisterCallback<ClickEvent>(_ =>
+            {
+                var targetWizard = _container.Q<VisualElement>("AddNewProfile");
+                ChangeWizard(targetWizard);
+            });
+        }
+
+        private void OpenLink(string url)
+        {
+            Application.OpenURL(url);
+        }
+
+        private void SetupTab()
+        {
+            SetupBootMenu();
+        }
+
+        private void SetupConfigSettings()
+        {
+            var selectedProfile = _stateManager.CoreApi.GetSetting(SettingsKeys.CurrentProfileName);
+            _stateManager.SelectedProfile = selectedProfile.Success ? selectedProfile.Value : _stateManager.AllProfiles.First();
+        }
+
+        private void SetupBootMenu()
```

Comment:
This method seems way more bloated then it needs to be. Consider something like the following (comments here are notes about this suggestion, not comments to be in the code):
```
/* Renamed SetupBootMenu */
private void ChooseProfileMenu()
{
    /* On initialization StateManager should be figuring out the profile to load */
    if (_stateManager.SelectedProfile == null)
    {
        ShowProfileMenu(createMenu);
    }
    else if (_stateManager.IsBootstrapped)
    {
        ShowProfileMenu(completedMenu);
    }
    else
    {
        ShowProfileMenu(bootstrapMenu);
    }
}

/* Renamed ChangeWizard */
private void ShowProfileMenu(VisualElement targetMenu)
{
  allMenus.ForEach(menu => menu.AddToClassList(hiddenClassName));
  if (targetMenu == null)
  {
    targetMenu.RemoveFromClassList(hiddenClassName);
  }
}
```